src/rpc/rawtransaction.cpp

```diff
@@ -906,6 +906,7 @@ UniValue sendrawtransaction(const JSONRPCRequest& request)
         }
         CValidationState state;
         bool fMissingInputs;
+        bool fLimitFree = true;
```

Comment:
This one is used. I slightly remember that this variable was somehow related to a vulnerability for which we backported a fix, and this fix removed that variable completely.

Reply:
I meant "is **un**used"

Reply:
removed in latest force push